tests/lib/snaps/store/test-snapd-with-default-configure-core18/meta/hooks/configure

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# Snapshot information
+infoFile="$SNAP_COMMON"/configure-info
+valueA=$( snapctl get a )
+valueB=$( snapctl get b )
+{
+echo "a: $valueA" 
```

Comment:
(nitpick) I would indent this with 2 or 4 spaces to make it slightly easier to read (but maybe this is preexisting in a different test case which case of course feel free to ignore)

Reply:
Done